### PR TITLE
gz-transport13: depend on gz-msgs10

### PR DESCRIPTION
**Pull request by j-rivero**
Remove the alias, copy the transport12 formula and bump dependency on msgs to version 10. Update all references to 12 to 13.

### CODE REVIEW

**Review by caguero — APPROVED** [review]
Not sure if it's needed but I've seen that we create an alias for the next version (`transport14`) in similar PRs...

**Comment by scpeters** [comment]
> Not sure if it's needed but I've seen that we create an alias for the next version (`transport14`) in similar PRs...

it's nice to do but not required

File: Formula/gz-transport13.rb [hunk 1]
```diff
@@ -0,0 +1,71 @@
+class Gztransport13 < Formula
```
Comment (scpeters) [inline Formula/gz-transport13.rb:1]: brew complaint: 

~~~
Expected to find class GzTransport13, but only found: Gztransport13.
~~~

Comment (caguero) [reply Formula/gz-transport13.rb]: [5f8c284](https://github.com/osrf/homebrew-simulation/pull/2196/commits/5f8c2842ce862bab4545de8af59f6157d1e225cc)

**Pull request merged**